tket2/src/circuit/extract_dfg.rs

```diff
@@ -89,8 +89,8 @@ fn remove_cfg_empty_output_tuple(
     let new_op = Output {
         types: new_types.clone().into(),
     };
-    let new_node = hugr.add_node_with_parent(
-        parent,
+    let new_node = hugr.add_node_after(
+        input_node,
```

Comment:
this should be covered right? the test _looks_ ok to me

Reply:
Turns out the "guppy-like" circuit was not guppy-like. I was loading a constant tag instead of using the `Tag` operation...

It should be fixed now.